dpnp/backend/kernels/dpnp_krnl_random.cpp

```diff
@@ -742,6 +742,76 @@ void dpnp_rng_standard_t_c(void* result, const _DataType df, const size_t size)
     }
 }
 
+template <typename _KernelNameSpecialization>
+class dpnp_rng_triangular_ration_acceptance_c_kernel;
+
+template <typename _DataType>
+void dpnp_rng_triangular_c(
+    void* result, const _DataType x_min, const _DataType x_mode, const _DataType x_max, const size_t size)
+{
+    if (!size)
+    {
+        return;
+    }
+    _DataType* result1 = reinterpret_cast<_DataType*>(result);
+    const _DataType d_zero = (_DataType(0));
+    const _DataType d_one = (_DataType(1));
```

Comment:
these variables are used once only. It is better to move values inplace.

Reply:
Thank you. Yes, it can be done on other PR for refactoring, where I will do the same changes for other funcs.